package.json

```diff
@@ -28,7 +28,10 @@
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-scripts": "^1.0.17",
-    "redis": "^2.8.0"
+    "redis": "^2.8.0",
+    "socket.io": "^2.0.4",
+    "socket.io-redis": "^5.2.0",
+    "sticky-session": "^1.1.2"
```

Comment:
https://qiita.com/takehilo/items/8c773d18ec6cfccd6679

Referred to this article. (Japanese article.)